chapters/annotations.tex

```diff
@@ -925,20 +925,19 @@ \subsection{Connections}\label{connections1}
 \end{example}
 
 The optional \lstinline!Text!\index{Text@\robustinline{Text}!\robustinline{connect} annotation} primitive defines a text that will be written on the connection line.
-It has the following definition (\emph{it is not equal to the \lstinline!Text! primitive as part of graphics -- the differences are marked as bold lines}):
-% NOTE: Technically just the names -- not the entire lines are marked in bold
+It has the following definition (\emph{it is not equal to the \lstinline!Text! primitive as part of graphics -- the differences are marked after Note in the description-strings}):
```

Comment:
Missing font markup: `Note` should be `\emph{Note}`.

Reply:
Actually, the use of `\emph` for the entire parenthesis seems wrong (nowadays we use either non-normative paragraphs (set in italic), or a parenthesis or wording (set in upright) to indicate that something is of lesser importance).  It should simply be this:
```
It has the following definition (it is not equal to the \lstinline!Text! primitive as part of graphics -- the differences are marked after \emph{Note} in the description-strings):
```

Reply:
A suggested fix is provided in #3377.